cmd/cluster-agent/app/app.go

```diff
@@ -184,7 +185,9 @@ func start(cmd *cobra.Command, args []string) error {
 
 	log.Infof("Datadog Cluster Agent is now running.")
 
-	apiCl, err := apiserver.GetAPIClient() // make sure we can connect to the apiserver
+	retrierCtx, cancel := context.WithTimeout(context.Background(), config.Datadog.GetDuration("kubernetes_apiserver_retrier_timeout")*time.Minute)
```

Comment:
The new parameter is not useful. The retrier in `GetAPIClient` is hardcoded to permafail after 5 minutes.

Reply:
PR updated 👌